### PR TITLE
Sort results in fixity report email

### PR DESCRIPTION
**Pull request by Dhwaniartefact**
Connected to https://github.com/archivematica/Issues/issues/1681

### CODE REVIEW

**Review by replaceafill — APPROVED** [review]
LGTM. Let's wait for @amayita's feedback before merging.

**Review by amayita — APPROVED** [review]
LGTM

**Pull request merged**
